src/client/sandbox/event/focus-blur.js

```diff
@@ -181,7 +148,36 @@ export default class FocusBlurSandbox extends SandboxBase {
 
         if (el[type]) {
             // NOTE: To guarantee that all focus/blur events are raised, we need to raise them manually.
-            this._ensureWindowScroll(el, preventScrolling, type, callback);
+            let windowScroll                           = null;
+            let elementNonScrollableParentsScrollState = null;
+
+            if (preventScrolling)
+                windowScroll = styleUtils.getElementScroll(this.window);
+
+            if (browserUtils.isIE)
+                elementNonScrollableParentsScrollState = this._getElementNonScrollableParentsScrollState(el);
+
+            if (this._shouldUseLabelHtmlForElement(el, type)) {
```

Comment:
I don't like this way.
It is not clear why `_ensureWindowScroll` method retruns `el` or `null`.